### ISSUE TITLE
NullReferenceException in AddTexturedQuad.Prefix

### ISSUE THREAD

**Issue by viktor-ferenczi** [WesternGamer/ProperGPSMarkerScaling#4]
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at ClientPlugin.Patches.MyHudMarkerRenderBase_AddTexturedQuad.Prefix(Single& halfWidth, Single& halfHeight)
   at Sandbox.Game.GUI.HudViewers.MyHudMarkerRenderBase.AddTexturedQuad_Patch1(MyHudMarkerRenderBase this, MyHudTexturesEnum texture, Vector2 position, Vector2 upVector, Color color, Single halfWidth, Single halfHeight)
   at Sandbox.Game.GUI.HudViewers.MyHudMarkerRender.PointOfInterest.Draw(MyHudMarkerRender renderer, Single alphaMultiplierMarker, Single alphaMultiplierText, Single scale, Boolean drawBox)
   at Sandbox.Game.GUI.HudViewers.MyHudMarkerRender.Draw()
   at Sandbox.Game.Gui.MyGuiScreenHudSpace.DrawAsync()
   at ParallelTasks.DelegateWork.DoWork(WorkData workData)
   at ParallelTasks.WorkItem.DoWork(Int32 expectedID)
```

**Comment by WesternSpace** [comment]
Exception was a result of an earlier exception, which did not set the global variables that the erroring code was trying to access.
Fixed in ca888baaae769a469ed6298e15667e7c7a215391

**Comment by viktor-ferenczi** [comment]
Perfect, thank you! Re-enabling your plugin.
